djstripe/event_handlers.py

```diff
@@ -200,8 +177,6 @@ def _handle_crud_like_event(
 	id=None,
 	customer=None,
 	crud_type=None,
-	crud_exact=False,
-	crud_valid=False,
```

Comment:
If we're OK with removing these params we could also probably remove crud_type as well since I don't think we're passing it in anywhere.

Reply:
Reviewing pr #858 and keeping issue #576 in mind, I'd leave crud_type in there so there's something handling the customer.subscription.deleted_oops_we_mean_updated edge-case.